### PR TITLE
Small changes in Network BG concepts....

### PR DESCRIPTION
**Pull request by mcwimm**
... in preparation for new network concepts

### CODE REVIEW

File: TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py [hunk 1]
```diff
@@ -241,18 +241,6 @@ def calculateTreeSalinity(self):
...
-    # Unit: kg per sec per cell volume
-    def calculateTreeContribution(self):
-        self.tree_contributions = np.zeros(len(self._salinity))
-        for tree_id in range(self.no_trees):
-            ids = self._tree_cell_ids[tree_id]
-            v = self.getVolume(affected_cells=ids)
-            per_volume = 1. / v
-            tree_contribution = self.tree_water_uptake[tree_id]
-            self.tree_contributions[ids] = tree_contribution * per_volume
-
```
Comment (jbathmann) [inline TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py:255]: why isn't this used here anymore?

File: TreeModelLib/BelowgroundCompetition/SimpleNetwork/SimpleNetwork.py [hunk 2]
```diff
@@ -142,8 +142,12 @@ def addTree(self, tree):
         self._psi_height.append(
             -(2 * geometry["r_crown"] + geometry["h_stem"]) * 9810)
         self._psi_top = np.array(self._psi_leaf) - np.array(self._psi_height)
-        self._psi_osmo = np.array(
-            [0] * self.n_trees)  # Salinity is 0 ppt is the basic scenario
+
+        # psi osmo is added separately in some below-ground competition concept
+        if self.bg_concept in ["SimpleNetwork", "NetworkFixedSalinity",
+                               "NetworkHydro"]:
```
Comment (jbathmann) [inline TreeModelLib/BelowgroundCompetition/SimpleNetwork/SimpleNetwork.py:148]: thi needs to be updated, whenever a new network concept is added? can you avoid this?

File: TreeModelLib/GrowthAndDeathDynamics/SimpleBettina/SimpleBettina.py [hunk 3]
```diff
@@ -161,19 +168,21 @@ def bgResources(self, belowground_resources):
...
     def deltaPsi(self):
         return (self.parameter["leaf_water_potential"] +
                 (2 * self.r_crown + self.h_stem) * 9810)
 
-    ## This function calculates the available resources and the biomass increment.
+    ## This function calculates the available resources and the biomass
+    # increment.
     def growthResources(self):
         self.available_resources = min(self.ag_resources, self.bg_resources)
         self.grow = (self.parameter["growth_factor"] *
```
Comment (jbathmann) [inline TreeModelLib/GrowthAndDeathDynamics/SimpleBettina/SimpleBettina.py:188]: only yapf in this file or did you change any functionality? :-)

File: TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py (hunk 1 above)
Comment (mcwimm) [reply TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py:254]: we don't need the total tree contribution in this concept, only constant contribution and salinity prefactor.

File: TreeModelLib/GrowthAndDeathDynamics/SimpleBettina/SimpleBettina.py (hunk 3 above)
Comment (mcwimm) [reply TreeModelLib/GrowthAndDeathDynamics/SimpleBettina/SimpleBettina.py:188]: yes, yapf

File: TreeModelLib/BelowgroundCompetition/SimpleNetwork/SimpleNetwork.py (hunk 2 above)
Comment (mcwimm) [reply TreeModelLib/BelowgroundCompetition/SimpleNetwork/SimpleNetwork.py]: yes. fixed :)

**Pull request merged**